src/combobox/combobox.module.ts

```diff
@@ -10,6 +10,7 @@ import { ComboBox } from "./combobox.component";
 import { DropdownModule } from "../dropdown/dropdown.module";
 import { I18nModule } from "./../i18n/i18n.module";
 import { UtilsModule } from "../utils/index";
+import { DropdownService } from "../dropdown/dropdown.service";
```

Comment:
Ideally we should import from the `index` for these ... it'll make the transition to entry points easier to manage